src/Control/CookieJar.php

```diff
@@ -180,4 +187,17 @@ protected function outputCookie(
         }
         return false;
     }
+
+    /**
+     * Get the correct samesite value - Session cookies use a different configuration variable.
+     *
+     * @deprecated 5.0 The relevant methods will include a `$sameSite` parameter instead.
```

Comment:
Deprecating this method is weird now the extension hook has been removed

Is there someway to 'deprecate' method signature on Cookie_Backed::set() to remind ourselves to add the extra parameter?

Reply:
We won't need this method once the attributes are added, because `Session::start()` will just pass the samesite value directly into its call to `Cookie::set()`.